packages/core-mobile/app.json

```diff
@@ -1,4 +1,16 @@
 {
   "name": "AvaxWallet",
-  "displayName": "Core X"
+  "displayName": "Core X",
+  "slug": "core-wallet",
```

Comment:
what is this slug used for?

Reply:
https://docs.expo.dev/versions/latest/config/app/#slug

it should be a url friendly name for our app, and it's required field in app.json. I believe expo uses this in their urls, like `https://expo.dev/@your-username/{slug}`, but I don't think we're exposing our app via expo.